src/main/java/com/codingParkFun/rmsbackend/controller/StudentController.java

```diff
@@ -28,5 +30,32 @@ public void createStudent( @RequestBody Student student ) {
         return studentService.get();
```

Comment:
It will return all the students from all the school

Reply:
No, all the students from the particular school.

Reply:
Shouldn't it will take school Id as argument?